test/src/e2e.test.ts

```diff
@@ -1129,7 +1129,7 @@ describe('EndToEnd', () => {
       const [res1, res2] = await Promise.allSettled([req1, req2])
       assert.equal(res1.status, 'fulfilled')
       assert.equal(res2.status, 'fulfilled')
-      assert.equal(res1.value.statusCode, 400)
+      // assert.equal(res1.value.statusCode, 400)
```

Comment:
This is an important e2e locking test. The test with all its assertions still works as expected except that the first request now succeeds with `200` instead of `400`. This might be blocking for the merge? 🤔